scripts/shared/clusters.sh

```diff
@@ -304,13 +302,28 @@ function warn_inotify() {
     fi
 }
 
+# If any of the clusters use OVN-K as the CNI then clone the
+# ovn-kubernetes repo from master in order to access the required
+# kind scripts, and manifest generation templates.
+function download_ovnk() {
+    if [[ ${cluster_cni[*]} != *"ovn"* ]]; then
+        return
+    fi
+
+    echo "Cloning ovn-kubernetes from source"
+    rm -rf ovn-kubernetes/
```

Comment:
I think part of @mkolesnik’s comment was that we shouldn’t unconditionally delete `ovn-kubernetes`, to avoid having to re-download the full git repository all the time when building locally. That’s how the suggested `git clone ... || git reset` works: if `ovn-kubernetes` already exists, `git clone` will fail, and instead `git fetch && git reset` will update the existing clone.

Reply:
Ah yep of course, sorry about this, fixed!